backend/redis/redis.go

```diff
@@ -34,6 +35,12 @@ type config struct {
 	PoolTimeout        time.Duration `env:"REDIS_POOL_TIMEOUT"`
 	IdleTimeout        time.Duration `env:"REDIS_IDLE_TIMEOUT"`
 	IdleCheckFrequency time.Duration `env:"REDIS_IDLE_CHECK_FREQUENCY"`
+	// Name of the key that is written to check, if redis is healthy
+	HealthKey string `env:"REDIS_HEALTH_KEY" envDefault:"healthy"`
+	// Amount of time to cache the last health check result
+	HealthMaxRequest time.Duration `env:"REDIS_HEALTHCHECK_MAX_REQUEST_SEC" envDefault:"10s"`
```

Comment:
Same as in `backend/postgres/postgres.go`: Think about streamlining naming here.